### PR TITLE
add documentation for subresource integrity

### PR DESCRIPTION
**Pull request by kumagi**
Add mentions to subresource integrity calculation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,3 +95,9 @@ http {
     }
 }
 ```
+
+### Subresource support
+
+This plugin embeds subresource integrity on generating SXG.
```
Comment (gaul) [inline README.md]: s/on/when/

File: README.md [hunk 2]
```diff
@@ -95,3 +95,9 @@ http {
     }
 }
 ```
+
+### Subresource support
+
```
Comment (gaul) [inline README.md]: Could we explain the user-facing behavior first then follow with the technical explanation, e.g.,

nginx-sxg-module automatically includes signatures of subresources in its responses, allowing prefetching.

File: README.md [hunk 3]
```diff
@@ -95,3 +95,9 @@ http {
...
+This plugin embeds subresource integrity on generating SXG.
+On finding `link: rel="preload"` entry in HTTP response header from upstream, this plugin will collect the specified resource to the upstream and append `rel="allowd-alt-sxg";header-integrity="sha256-...."` to the original HTTP response automatically.
```
Comment (gaul) [inline README.md]: s/On/When/

Typo: allow**e**d?

File: README.md (hunk 1 above)
Comment (kumagi) [reply README.md]: done.

File: README.md (hunk 2 above)
Comment (kumagi) [reply README.md]: nice suggestion.

File: README.md (hunk 3 above)
Comment (kumagi) [reply README.md]: Done.

**Comment by kumagi** [comment]
Thank you for quick response!

**Pull request merged**